modules/tinymce/src/plugins/table/main/ts/ui/CellDialog.ts

```diff
@@ -141,10 +142,9 @@ const open = (editor: Editor) => {
       }
     ],
     initialData: data,
-    onSubmit: Fun.curry(onSubmitCellForm, editor, cells)
+    onSubmit: Fun.curry(onSubmitCellForm, editor, Arr.map(cells, (c) => c.dom()))
```

Comment:
I ended up doing this in a couple of places and I'm not overly fond of it, but not sure what the best alternative would be. I assume `TableSelection.getCellsFromSelection` returns Sugar Elements for a reason so... Could map it to DOM Elements earlier? Thoughts?

Reply:
Snooker/Darwin, definitely should be using sugar elements. We could maybe add a wrapper within the plugin if needed? Another option would be to update more of the plugin to use sugar elements.